mmdet/models/detectors/retinanet.py

```diff
@@ -1,9 +1,10 @@
 from ..builder import DETECTORS
 from .single_stage import SingleStageDetector
+from .single_stage_test_mixins import BBoxTestMixin
 
 
 @DETECTORS.register_module()
-class RetinaNet(SingleStageDetector):
+class RetinaNet(SingleStageDetector, BBoxTestMixin):
```

Comment:
To be consistent with two-stage detectors, it is better to make BBoxTestMixin a mixin of DenseHead instead of Detector.

Reply:
I have updated codes.